core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -304,7 +303,8 @@ class KenyaCodes(object):
     SOUTH_MUGIRANGO = "south mugirango"
     SOY = "soy"
     STAREHE = "starehe"
-    SUBA = "suba"
+    SUBA_NORTH_MBITA = "suba north/mbita"
```

Comment:
Is this a renamed constituency or is it now officially known as "Suba North/Mbita"?

If it was renamed and is now only referred to as "Suba North" officially, then I suggest dropping the "Mbita".

Reply:
It was renamed and is now officially referred to as "Suba North". I've dropped "Mbita".